hwbench/environment/test_parse.py

```diff
@@ -275,6 +275,19 @@ def test_nvme_parsing_version_v116(self):
         assert version == (d / "version").read_bytes().strip()
 
 
+class TestParseSdparm:
+    def test_sdparm_parsing_version_v110(self):
```

Comment:
you are missing a test to validated the parsing of sdparm's output, you are only parsing the version.